src/nbl/video/device_capabilities/device_limits.json

```diff
@@ -1245,152 +1243,152 @@
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "integerDotProductAccumulatingSaturating4x8BitPackedUnsignedAccelerated",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "integerDotProductAccumulatingSaturating4x8BitPackedSignedAccelerated",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "integerDotProductAccumulatingSaturating4x8BitPackedMixedSignednessAccelerated",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "integerDotProductAccumulatingSaturating16BitUnsignedAccelerated",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "integerDotProductAccumulatingSaturating16BitSignedAccelerated",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "integerDotProductAccumulatingSaturating16BitMixedSignednessAccelerated",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "integerDotProductAccumulatingSaturating32BitUnsignedAccelerated",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "integerDotProductAccumulatingSaturating32BitSignedAccelerated",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "integerDotProductAccumulatingSaturating32BitMixedSignednessAccelerated",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "integerDotProductAccumulatingSaturating64BitUnsignedAccelerated",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "integerDotProductAccumulatingSaturating64BitSignedAccelerated",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "integerDotProductAccumulatingSaturating64BitMixedSignednessAccelerated",
-                    "value": false
+                    "value": true
```

Comment:
if you were to promote them, then you'd need `expose` tag, I'd suggest to leave as is right now, its difficult to roll back requirements